backend/src/main/java/fk/prof/backend/aggregator/AggregationWindow.java

```diff
@@ -35,9 +42,15 @@ public AggregationWindow(String appId, String clusterId, String procId,
 
     Map<Long, ProfileWorkInfo> workInfoModifiableLookup = new HashMap<>();
     for (int i = 0; i < workIds.length; i++) {
-      workInfoModifiableLookup.put(workIds[i], new ProfileWorkInfo());
+      workInfoModifiableLookup.put(workIds[i], new ProfileWorkInfo(workDurationInSec));
     }
     this.workInfoLookup = Collections.unmodifiableMap(workInfoModifiableLookup);
+
+    ProcessGroupTag processGroupTag = new ProcessGroupTag(appId, clusterId, procId);
+    this.mtrStartFailures = metricRegistry.meter(MetricRegistry.name(AggregationWindow.class, "start", "fail", processGroupTag.toString()));
```

Comment:
I think single meter to capture update failure rate would suffice. Anyway these failure are actually exceptions irl and will not happen when things are stable and when they do happen we are anyway logging things.